ovos_config/mycroft.conf

```diff
@@ -487,10 +487,10 @@
     },
 
     // Microphone plugin to be read audio
-    // NOTE: only in ovos-dinkum-listener
     "microphone": {
-      // use "ovos-microphone-plugin-sounddevice" for MacOS support
-      "module": "ovos-microphone-plugin-alsa"
+      "module": "ovos-microphone-plugin-sounddevice",
```

Comment:
potentially a breaking change, for setups running alsa that update without https://github.com/OpenVoiceOS/ovos-plugin-manager/pull/263

Reply:
https://github.com/OpenVoiceOS/ovos-config/pull/155